### ISSUE TITLE
bullet game

### ISSUE THREAD

**Issue by dharmendra1104** [Mayank-Garg7/BulletGame#4]
in your bullet game repository you have make class and object can you please describe me more about it.

**Comment by Mayank-Garg7** [comment]
> in your bullet game repository you have make class and object can you please describe me more about it.


Yes, why not. In this game I have used java script and used oops concept. Class and Object both are related to the Oops concept.
